examples/complete-cloudtrail/main.tf

```
@@ -1,15 +1,24 @@
-provider "lacework" {}
-
-provider "aws" {
-  region = "us-west-2"
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+      version = "~> 0.2.7"
+    }
+  }
 }
 
+provider "aws" {}
+
+provider "lacework" {}
+
 module "aws_cloudtrail" {
-  source = "../../"
+  source  = "lacework/cloudtrail/aws"
+  version = "~> 0.1.3"
```

These are living tests, we need them to point to the local directory:
```suggestion
  source = "../../"
```